qa/suites/upgrade/client-upgrade-luminous/luminous-client-x/basic/2-workload/rbd_cli_import_export.yaml

```diff
@@ -0,0 +1,13 @@
+tasks:
+- sequential:
+  - upgrade_workload
+- ceph: 
+- print: "**** done ceph"
+- workunit:
+    branch: luminous
+    clients:
+      client.0:
+      - rbd/import_export.sh
+    env:
```

Comment:
Nit: drop this line and the next

Reply:
removed:
    env:
      RBD_CREATE_ARGS: --image-feature layering,exclusive-lock,object-map